src/sync.js

```diff
@@ -877,6 +889,12 @@ SyncApi.prototype._onSyncError = function(err, syncOptions) {
     console.error("/sync error %s", err);
     console.error(err);
 
+    if (err.errcode === "M_UNKNOWN_TOKEN") {
```

Comment:
avoid duplication with `if (shouldAbortSync(err)) { return; }` here?

Reply:
This is a different function, but there's nothing stopping the function from being defined in a higher scope I guess.